static/js/message_live_update.js

```diff
@@ -14,14 +14,26 @@ function rerender_messages_view() {
     }
 }
 
+function rerender_messages_view_for_user(user_id) {
+    for (const list of [message_lists.home, message_list.narrowed]) {
+        if (list === undefined) {
+            continue;
+        }
+        if (list.table_name !== undefined) {
+            const msgs = list.data.get_messages_sent_by_user(user_id);
```

Comment:
We are hoping to ban `msgs` as a variable name from the codebase; spell this `messages`.